src/lib/config/arbitrum/index.ts

```diff
@@ -0,0 +1,69 @@
+import keys from './keys';
+import contracts from './contracts';
+import pools from './pools';
+
+export default {
```

Comment:
I like the default because we want to ensure these are kept as config files, we don't want to be adding functions and other jazz to them, they are supposed to be equivalent to the JSON files as before just a little more flexible.